static/css/house_create.css

```diff
@@ -1,13 +1,14 @@
-#house_create{
-    align-content: center;
-    margin-top: 5%;
-    margin-bottom: auto;
+#house_create {
+  align-content: center;
+  margin-top: 5%;
+  margin-bottom: auto;
 }
 
-#submit-button{
+#submit-button {
   width: 100%;
+
 }
 
 select {
   display: block !important;
-}
+}
```

Comment:
missing line

Reply:
In css files the formating is without a new line I think